scripts/sct_download_data.py

```diff
@@ -51,6 +43,7 @@ def get_parser():
             'optic_models',
             'pmj_models',
             'binaries_debian',
+            'binaries_centos',
```

Comment:
This was gone in #2699.

Reply:
fixed in 190acb6bb392c179a7e40e2ea7e5e6f58cfd10b1